fix issues #609, #614, #629, #630 and new feature for dismiss downloads items

heliomarpm opened the pull request:
Thanks for contributing!

Please be sure you are following the guidelines at 
https://github.com/FaisalUmair/udemy-downloader-gui/blob/master/CONTRIBUTING.md

So sorry for my English.
# This version (1.8.5) contains: 

   * A new feature for dispensing items in the downloads section.
   * A new feature for Setting default subtitle
   * Translate popup for Update question
   * Option to open the selected course in the Udemy subdomain.
   * Fixed issue #609.
   * Fixed issue #614.
   * Fixed issue #629.
   * Fixed issue #630
   
 # download version for windows 64:
 https://github.com/heliomarpm/udemy-downloader-gui/releases/download/v1.8.5/Udeler-Setup-1.8.5-windows-x64-unofficial.exe

![](https://github.com/heliomarpm/udemy-downloader-gui/releases/download/v1.8.5/dispensar.png)

![](https://github.com/heliomarpm/udemy-downloader-gui/releases/download/v1.8.5/set.default.subtitle.png)

heliomarpm committed:
Add launch debug file

heliomarpm committed:
add launch debug file

heliomarpm committed:
hotfix issue #609 and new feature dismiss downloads items

heliomarpm committed:
fix: download attachements

heliomarpm committed:
translate update question & set default subtitle

ScrubDonkey commented:
Doesn't work.....While you can log in and begin download downloads are not complete working files.

heliomarpm committed:
added download completion date

heliomarpm committed:
added message download complete

heliomarpm committed:
format index.html

heliomarpm committed:
new icon and url repository

heliomarpm committed:
Added course indicator with encrypted classes

heliomarpm committed:
Download completed notification update

heliomarpm committed:
update icon

heliomarpm committed:
new options in settings

heliomarpm committed:
added hot-reload

heliomarpm committed:
new colors

heliomarpm committed:
commit v1.8.7

heliomarpm committed:
Update README.md

heliomarpm committed:
Update README.md

heliomarpm committed:
Update README.md

heliomarpm committed:
Update README.md

heliomarpm committed:
Update README.md

heliomarpm committed:
Update README.md

heliomarpm committed:
Update README.md

heliomarpm committed:
Update README.md

heliomarpm committed:
Update README.md

heliomarpm committed:
Update README.md

heliomarpm committed:
ajuste mensagens

heliomarpm committed:
update locale ru.json

heliomarpm committed:
Update README.md

heliomarpm committed:
Create build.yml

heliomarpm committed:
fix message complete

heliomarpm committed:
Merge branch 'master' of https://github.com/heliomarpm/udemy-downloader-gui

heliomarpm committed:
up version 190

heliomarpm committed:
remove package.lock

heliomarpm committed:
remove action builder

heliomarpm committed:
update packages

heliomarpm committed:
gitaction

heliomarpm committed:
gitactions

heliomarpm committed:
remove build

heliomarpm committed:
add gitaction build

heliomarpm committed:
gitaction build

heliomarpm committed:
Update build.yml

heliomarpm committed:
Update build.yml

heliomarpm committed:
Update build.yml

heliomarpm committed:
Update build.yml

heliomarpm committed:
v1.9.1

heliomarpm committed:
v1.9.1

heliomarpm committed:
Merge branch 'master' of https://github.com/heliomarpm/udemy-downloader-gui

heliomarpm committed:
Update README.md

heliomarpm committed:
v1.9.1

heliomarpm committed:
v1.9.1

heliomarpm committed:
v1.9.2

heliomarpm committed:
Merge branch 'master' of https://github.com/heliomarpm/udemy-downloader-gui

heliomarpm committed:
Update README.md

heliomarpm committed:
Ignore anexos já baixado

heliomarpm committed:
add option open download folder

heliomarpm committed:
remove flag encriptado em posterior download

heliomarpm committed:
v1.10

Version 1.10.1

* Option for open downloading folder
* New option config - Number download with zero left.
* Auto ignore attachments downloaded
* +Bugfixed

heliomarpm committed:
v1.10.1

heliomarpm committed:
v1.10.1

heliomarpm committed:
update readme

heliomarpm committed:
hotfix login with token

heliomarpm committed:
v1.10.2 - Fixed Login with Credentials

heliomarpm committed:
v1.10.3 - hotfix

heliomarpm committed:
Create codeql-analysis.yml

heliomarpm committed:
v1.11 - modify alert msg drm protected

heliomarpm committed:
Merge branch 'master' of https://github.com/heliomarpm/udemy-downloader-gui

heliomarpm committed:
v1.11 - update packages

heliomarpm committed:
v1.11 - remove tag html title

heliomarpm committed:
bugfix freezing in build course data

heliomarpm committed:
update changelog

heliomarpm committed:
Update issue templates

heliomarpm committed:
Update issue templates

heliomarpm committed:
Update FUNDING.yml

heliomarpm committed:
Update README.md

heliomarpm committed:
Update README.md

heliomarpm committed:
v1.11.2

heliomarpm committed:
Update build.yml

heliomarpm committed:
Create SECURITY.md

heliomarpm committed:
Update README.md

heliomarpm committed:
v1.11.2

heliomarpm committed:
Merge branch 'master' of https://github.com/heliomarpm/udemy-downloader-gui

heliomarpm committed:
add prebuild script for auto-increment version

heliomarpm committed:
add templates issues & pull requests

heliomarpm committed:
update template issues

Commit:
Set video qualitity as an empty string so it wont crash when the user tries to download in line 516

heliomarpm committed:
Merge pull request #56 from freddygarcia/bugfix/fix-stuck-on-building-course-data

Set video qualitity as an empty string so it wont crash when the user…

heliomarpm committed:
Update app.js

heliomarpm committed:
1.11.3 bugfixes

dependabot[bot] committed:
Bump moment from 2.29.1 to 2.29.4

Bumps [moment](https://github.com/moment/moment) from 2.29.1 to 2.29.4.
- [Release notes](https://github.com/moment/moment/releases)
- [Changelog](https://github.com/moment/moment/blob/develop/CHANGELOG.md)
- [Commits](https://github.com/moment/moment/compare/2.29.1...2.29.4)

---
updated-dependencies:
- dependency-name: moment
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

heliomarpm committed:
Update README.md

heliomarpm committed:
update docs

heliomarpm committed:
Merge branch 'master' of https://github.com/heliomarpm/udemy-downloader-gui

heliomarpm committed:
Update FUNDING.yml

heliomarpm committed:
Merge pull request #58 from heliomarpm/dependabot/npm_and_yarn/moment-2.29.4

Bump moment from 2.29.1 to 2.29.4

heliomarpm committed:
Update FUNDING.yml

heliomarpm committed:
update contribuidores

dependabot[bot] committed:
Bump socket.io-parser from 4.0.4 to 4.0.5

Bumps [socket.io-parser](https://github.com/socketio/socket.io-parser) from 4.0.4 to 4.0.5.
- [Release notes](https://github.com/socketio/socket.io-parser/releases)
- [Changelog](https://github.com/socketio/socket.io-parser/blob/main/CHANGELOG.md)
- [Commits](https://github.com/socketio/socket.io-parser/compare/4.0.4...4.0.5)

---
updated-dependencies:
- dependency-name: socket.io-parser
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump electron from 8.2.4 to 18.3.7

Bumps [electron](https://github.com/electron/electron) from 8.2.4 to 18.3.7.
- [Release notes](https://github.com/electron/electron/releases)
- [Changelog](https://github.com/electron/electron/blob/main/docs/breaking-changes.md)
- [Commits](https://github.com/electron/electron/compare/v8.2.4...v18.3.7)

---
updated-dependencies:
- dependency-name: electron
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>

heliomarpm committed:
Update README.md

dependabot[bot] committed:
Bump qs from 6.5.2 to 6.5.3

Bumps [qs](https://github.com/ljharb/qs) from 6.5.2 to 6.5.3.
- [Release notes](https://github.com/ljharb/qs/releases)
- [Changelog](https://github.com/ljharb/qs/blob/main/CHANGELOG.md)
- [Commits](https://github.com/ljharb/qs/compare/v6.5.2...v6.5.3)

---
updated-dependencies:
- dependency-name: qs
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

CetinOzdil committed:
Added high quality stream download option

CetinOzdil committed:
cleanups

CetinOzdil committed:
file renaming format changed

CetinOzdil committed:
updated build scripts

heliomarpm committed:
Merge pull request #87 from CetinOzdil/master

Allow downloading video streams (M3U8)

heliomarpm committed:
Merge pull request #78 from heliomarpm/dependabot/npm_and_yarn/socket.io-parser-4.0.5

Bump socket.io-parser from 4.0.4 to 4.0.5

heliomarpm committed:
Merge pull request #79 from heliomarpm/dependabot/npm_and_yarn/electron-18.3.7

Bump electron from 8.2.4 to 18.3.7

dependabot[bot] committed:
Bump minimist from 1.2.5 to 1.2.7

Bumps [minimist](https://github.com/minimistjs/minimist) from 1.2.5 to 1.2.7.
- [Release notes](https://github.com/minimistjs/minimist/releases)
- [Changelog](https://github.com/minimistjs/minimist/blob/main/CHANGELOG.md)
- [Commits](https://github.com/minimistjs/minimist/compare/v1.2.5...v1.2.7)

---
updated-dependencies:
- dependency-name: minimist
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

heliomarpm committed:
Merge pull request #84 from heliomarpm/dependabot/npm_and_yarn/qs-6.5.3

Bump qs from 6.5.2 to 6.5.3

dependabot[bot] committed:
Bump engine.io and socket.io

Bumps [engine.io](https://github.com/socketio/engine.io) to 6.2.1 and updates ancestor dependency [socket.io](https://github.com/socketio/socket.io). These dependencies need to be updated together.


Updates `engine.io` from 6.1.3 to 6.2.1
- [Release notes](https://github.com/socketio/engine.io/releases)
- [Changelog](https://github.com/socketio/engine.io/blob/main/CHANGELOG.md)
- [Commits](https://github.com/socketio/engine.io/compare/6.1.3...6.2.1)

Updates `socket.io` from 4.4.1 to 4.5.4
- [Release notes](https://github.com/socketio/socket.io/releases)
- [Changelog](https://github.com/socketio/socket.io/blob/main/CHANGELOG.md)
- [Commits](https://github.com/socketio/socket.io/compare/4.4.1...4.5.4)

---
updated-dependencies:
- dependency-name: engine.io
  dependency-type: indirect
- dependency-name: socket.io
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

heliomarpm committed:
Merge pull request #88 from heliomarpm/dependabot/npm_and_yarn/minimist-1.2.7

Bump minimist from 1.2.5 to 1.2.7

heliomarpm committed:
Merge pull request #89 from heliomarpm/dependabot/npm_and_yarn/engine.io-and-socket.io-6.2.1

Bump engine.io and socket.io

heliomarpm committed:
Update bug_report.yml

heliomarpm committed:
Update bug_report.yml

heliomarpm committed:
v1.11.5 - add quality 1080

heliomarpm committed:
update change_log

heliomarpm committed:
Update codeql-analysis.yml

heliomarpm committed:
Create release.yml

heliomarpm committed:
Update release.yml

heliomarpm committed:
v1.11.6

heliomarpm committed:
Merge branch 'master' of https://github.com/heliomarpm/udemy-downloader-gui

heliomarpm committed:
Update FUNDING.yml

heliomarpm committed:
v1.11.7

heliomarpm committed:
v1.11.7

heliomarpm committed:
v1.11.8

heliomarpm committed:
v1.11.9

heliomarpm committed:
v1.11.9

heliomarpm committed:
v1.11.10

heliomarpm committed:
v1.11.10

heliomarpm committed:
update readme & docs

heliomarpm committed:
update readme

heliomarpm committed:
update readme

heliomarpm committed:
update readme

heliomarpm committed:
update readme

heliomarpm committed:
update readme

heliomarpm committed:
update readme

heliomarpm committed:
update readme

heliomarpm committed:
update readme

heliomarpm committed:
update readme

heliomarpm committed:
update CHANGE_LOG

heliomarpm committed:
v1.12.0

heliomarpm committed:
v1.12.1

heliomarpm committed:
update workflow

dependabot[bot] committed:
Bump minimatch and electron-builder

Bumps [minimatch](https://github.com/isaacs/minimatch) to 5.1.6 and updates ancestor dependency [electron-builder](https://github.com/electron-userland/electron-builder/tree/HEAD/packages/electron-builder). These dependencies need to be updated together.


Updates `minimatch` from 3.0.4 to 5.1.6
- [Release notes](https://github.com/isaacs/minimatch/releases)
- [Changelog](https://github.com/isaacs/minimatch/blob/main/changelog.md)
- [Commits](https://github.com/isaacs/minimatch/compare/v3.0.4...v5.1.6)

Updates `electron-builder` from 23.6.0 to 24.0.0
- [Release notes](https://github.com/electron-userland/electron-builder/releases)
- [Changelog](https://github.com/electron-userland/electron-builder/blob/master/packages/electron-builder/CHANGELOG.md)
- [Commits](https://github.com/electron-userland/electron-builder/commits/v24.0.0/packages/electron-builder)

---
updated-dependencies:
- dependency-name: minimatch
  dependency-type: indirect
- dependency-name: electron-builder
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>

heliomarpm committed:
Merge pull request #101 from heliomarpm/dependabot/npm_and_yarn/minimatch-and-electron-builder-5.1.6

Bump minimatch and electron-builder

heliomarpm committed:
Update README.md

heliomarpm committed:
feat: badge logger & reduce sentry logger

heliomarpm committed:
Merge branch 'master' of https://github.com/heliomarpm/udemy-downloader-gui

AlAswaad99 committed:
feat: Amharic Locale Added

heliomarpm committed:
Merge pull request #112 from AlAswaad99/amharic_locale_added

feat: Amharic Locale Added

heliomarpm committed:
v1.12.2

heliomarpm committed:
Merge branch 'master' of https://github.com/heliomarpm/udemy-downloader-gui

heliomarpm committed:
v1.12.2

heliomarpm committed:
v1.12.3

heliomarpm committed:
update changelog

heliomarpm committed:
Update README.md

heliomarpm committed:
update changelog

heliomarpm committed:
v1.12.4

heliomarpm committed:
Update bug_report.yml

heliomarpm committed:
Update bug_report.yml

heliomarpm committed:
Update bug_report.yml

heliomarpm committed:
Merge pull request #124 from heliomarpm/heliomarpm-patch-1

hotfix bug_report.yml

heliomarpm committed:
chore: add .editorconfig, .prettierrc.yml

heliomarpm committed:
Merge branch 'master' of https://github.com/heliomarpm/udemy-downloader-gui

heliomarpm committed:
v1.12.5

heliomarpm committed:
update changelog

heliomarpm committed:
update changelog

heliomarpm committed:
deepscan clear code

heliomarpm committed:
v1.12.6

heliomarpm committed:
Update README.md

heliomarpm committed:
Merge pull request #136 from heliomarpm/heliomarpm-patch-1

Update README.md

heliomarpm committed:
Update feature_request.yml

heliomarpm committed:
Merge pull request #137 from heliomarpm/heliomarpm-patch-1

Update feature_request.yml

heliomarpm committed:
Update bug_report.yml

heliomarpm committed:
v1.12.7

mirusu400 committed:
Sync locale files with template.json

mirusu400 committed:
Change locale indent

mirusu400 committed:
Update ko.json

heliomarpm committed:
Merge pull request #139 from mirusu400/update-locale-template

Sync locale files with template.json

Thanks @mirusu400

mirusu400 committed:
Merge branch 'korean' of https://github.com/mirusu400/udemy-downloader-gui into korean

heliomarpm committed:
Merge pull request #140 from mirusu400/korean

Update korean translation

dependabot[bot] committed:
Bump word-wrap from 1.2.3 to 1.2.4

Bumps [word-wrap](https://github.com/jonschlinkert/word-wrap) from 1.2.3 to 1.2.4.
- [Release notes](https://github.com/jonschlinkert/word-wrap/releases)
- [Commits](https://github.com/jonschlinkert/word-wrap/compare/1.2.3...1.2.4)

---
updated-dependencies:
- dependency-name: word-wrap
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

heliomarpm committed:
Merge pull request #142 from heliomarpm/dependabot/npm_and_yarn/word-wrap-1.2.4

Bump word-wrap from 1.2.3 to 1.2.4

heliomarpm committed:
chore: update packages sentry, axios, jquery, electron_builder. open, sem_ver

heliomarpm committed:
Update config.yml

heliomarpm committed:
update issues templates

heliomarpm committed:
fix: download speed information

heliomarpm committed:
update change_log

kafuikpoh committed:
Rename file to align with instructions in docs

heliomarpm committed:
Merge pull request #149 from kafuikpoh/master

Rename file to align with instructions in docs

snyk-bot committed:
fix: upgrade axios from 1.4.0 to 1.5.0

Snyk has created this PR to upgrade axios from 1.4.0 to 1.5.0.

See this package in npm:
https://www.npmjs.com/package/axios

See this project in Snyk:
https://app.snyk.io/org/heliomarpm/project/aa69d961-2a47-4212-a08b-512fa7ca3272?utm_source=github&utm_medium=referral&page=upgrade-pr

snyk-bot committed:
fix: upgrade jquery from 3.7.0 to 3.7.1

Snyk has created this PR to upgrade jquery from 3.7.0 to 3.7.1.

See this package in npm:
https://www.npmjs.com/package/jquery

See this project in Snyk:
https://app.snyk.io/org/heliomarpm/project/aa69d961-2a47-4212-a08b-512fa7ca3272?utm_source=github&utm_medium=referral&page=upgrade-pr

heliomarpm committed:
Merge pull request #154 from heliomarpm/snyk-upgrade-123e3bc6e409638a186b3109b3a3df4e

[Snyk] Upgrade axios from 1.4.0 to 1.5.0

heliomarpm committed:
Merge pull request #155 from heliomarpm/snyk-upgrade-925abb7688c0be6f4e600daf2f673dfe

[Snyk] Upgrade jquery from 3.7.0 to 3.7.1

snyk-bot committed:
fix: upgrade @sentry/electron from 4.10.0 to 4.11.0

Snyk has created this PR to upgrade @sentry/electron from 4.10.0 to 4.11.0.

See this package in npm:
https://www.npmjs.com/package/@sentry/electron

See this project in Snyk:
https://app.snyk.io/org/heliomarpm/project/aa69d961-2a47-4212-a08b-512fa7ca3272?utm_source=github&utm_medium=referral&page=upgrade-pr

heliomarpm committed:
Merge pull request #157 from heliomarpm/snyk-upgrade-bc3505ad2700c90d53eb88ac7bdafb41

[Snyk] Upgrade @sentry/electron from 4.10.0 to 4.11.0

snyk-bot committed:
fix: upgrade axios from 1.5.0 to 1.5.1

Snyk has created this PR to upgrade axios from 1.5.0 to 1.5.1.

See this package in npm:
https://www.npmjs.com/package/axios

See this project in Snyk:
https://app.snyk.io/org/heliomarpm/project/aa69d961-2a47-4212-a08b-512fa7ca3272?utm_source=github&utm_medium=referral&page=upgrade-pr

heliomarpm committed:
Merge pull request #161 from heliomarpm/snyk-upgrade-6c0d6f947683cdc263e8d1cc1546ea57

[Snyk] Upgrade axios from 1.5.0 to 1.5.1

snyk-bot committed:
fix: upgrade @sentry/electron from 4.11.0 to 4.11.1

Snyk has created this PR to upgrade @sentry/electron from 4.11.0 to 4.11.1.

See this package in npm:
https://www.npmjs.com/package/@sentry/electron

See this project in Snyk:
https://app.snyk.io/org/heliomarpm/project/aa69d961-2a47-4212-a08b-512fa7ca3272?utm_source=github&utm_medium=referral&page=upgrade-pr

heliomarpm committed:
Merge pull request #162 from heliomarpm/snyk-upgrade-b44f879a4c03b1375c3cbaf96d397adb

[Snyk] Upgrade @sentry/electron from 4.11.0 to 4.11.1

snyk-bot committed:
fix: upgrade @sentry/electron from 4.11.1 to 4.12.0

Snyk has created this PR to upgrade @sentry/electron from 4.11.1 to 4.12.0.

See this package in npm:
https://www.npmjs.com/package/@sentry/electron

See this project in Snyk:
https://app.snyk.io/org/heliomarpm/project/aa69d961-2a47-4212-a08b-512fa7ca3272?utm_source=github&utm_medium=referral&page=upgrade-pr

heliomarpm committed:
Merge pull request #163 from heliomarpm/snyk-upgrade-e578811b4ca5e2b8c6c3cbc2733a3ca1

[Snyk] Upgrade @sentry/electron from 4.11.1 to 4.12.0

snyk-bot committed:
fix: upgrade @sentry/electron from 4.12.0 to 4.13.0

Snyk has created this PR to upgrade @sentry/electron from 4.12.0 to 4.13.0.

See this package in npm:
https://www.npmjs.com/package/@sentry/electron

See this project in Snyk:
https://app.snyk.io/org/heliomarpm/project/aa69d961-2a47-4212-a08b-512fa7ca3272?utm_source=github&utm_medium=referral&page=upgrade-pr

heliomarpm committed:
Merge pull request #164 from heliomarpm/snyk-upgrade-0e3def163c1d601866f2106bff6fda0f

[Snyk] Upgrade @sentry/electron from 4.12.0 to 4.13.0

snyk-bot committed:
fix: upgrade @sentry/electron from 4.13.0 to 4.14.0

Snyk has created this PR to upgrade @sentry/electron from 4.13.0 to 4.14.0.

See this package in npm:
https://www.npmjs.com/package/@sentry/electron

See this project in Snyk:
https://app.snyk.io/org/heliomarpm/project/aa69d961-2a47-4212-a08b-512fa7ca3272?utm_source=github&utm_medium=referral&page=upgrade-pr

heliomarpm committed:
Merge pull request #166 from heliomarpm/snyk-upgrade-87537a072c6946c30e34cc407af2a736

[Snyk] Upgrade @sentry/electron from 4.13.0 to 4.14.0

snyk-bot committed:
fix: upgrade cookie from 0.5.0 to 0.6.0

Snyk has created this PR to upgrade cookie from 0.5.0 to 0.6.0.

See this package in npm:
https://www.npmjs.com/package/cookie

See this project in Snyk:
https://app.snyk.io/org/heliomarpm/project/aa69d961-2a47-4212-a08b-512fa7ca3272?utm_source=github&utm_medium=referral&page=upgrade-pr

dependabot[bot] committed:
Bump app-builder-lib and electron-builder

Bumps [app-builder-lib](https://github.com/electron-userland/electron-builder/tree/HEAD/packages/app-builder-lib) to 24.13.3 and updates ancestor dependency [electron-builder](https://github.com/electron-userland/electron-builder/tree/HEAD/packages/electron-builder). These dependencies need to be updated together.


Updates `app-builder-lib` from 24.6.3 to 24.13.3
- [Release notes](https://github.com/electron-userland/electron-builder/releases)
- [Changelog](https://github.com/electron-userland/electron-builder/blob/master/packages/app-builder-lib/CHANGELOG.md)
- [Commits](https://github.com/electron-userland/electron-builder/commits/v24.13.3/packages/app-builder-lib)

Updates `electron-builder` from 24.6.3 to 24.13.3
- [Release notes](https://github.com/electron-userland/electron-builder/releases)
- [Changelog](https://github.com/electron-userland/electron-builder/blob/master/packages/electron-builder/CHANGELOG.md)
- [Commits](https://github.com/electron-userland/electron-builder/commits/v24.13.3/packages/electron-builder)

---
updated-dependencies:
- dependency-name: app-builder-lib
  dependency-type: indirect
- dependency-name: electron-builder
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>

heliomarpm committed:
Merge pull request #171 from heliomarpm/snyk-upgrade-30e4581a0e68b04159202b58700f7a18

[Snyk] Upgrade cookie from 0.5.0 to 0.6.0

heliomarpm committed:
Merge pull request #179 from heliomarpm/dependabot/npm_and_yarn/app-builder-lib-and-electron-builder-24.13.3

Bump app-builder-lib and electron-builder
